ReviewsReceiver/src/models/reviewsModel.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose from "mongoose";
+
+export interface IReview {
+  businessId: string;
```

Comment:
Just a thought maybe in the future we will need a service for generating ID because it's a distributed ID that we should use and address in every component